**Glossary**

***Alpha helix*** - A type of secondary structure seen in proteins that is characterized by a regular pattern of hydrogen bonds between local regions of the protein chain (carbonyl group of one amino acid and the amino group of the amino acid 4 residues below it along the helix). The backbone atoms of the peptide in this region forms a right handed helical structure, hence the name.

***Angstrom*** - A unit of length equal to 10-10 meters. It is the approximate diameter of a hydrogen atom and is commonly used to measure atomic dimensions.

***Asymmetric unit*** - The smallest part of a crystal structure to which space group symmetry can be applied to produce the entire crystal. The asymmetric unit may contain a whole molecule, a portion of a molecule, or multiple molecules and does not necessarily represent the functional unit of the molecule under investigation.

***Atom*** - The smallest unit of a chemical element that has the characteristics of that element. An atom is composed of protons and neutrons in a nucleus that is surrounded by electrons.

***Beta sheet*** - A type of secondary structure seen in proteins that is characterized by a regular pattern of hydrogen bonds between 2 or more strands of amino acid chains lying side-by-side running either parallel to each other or antiparallel. The hydrogen bonds are formed between carbonyl groups of one chain and amino groups of the adjacent chain to form a sheet-like structure.

***Biological assembly*** - The biologically functional molecule which may be different from the crystallographic asymmetric unit (see *asymmetric unit*).

***Cartesian coordinates*** - An X-Y-Z coordinate system with the axes at 90 degrees to each other.

***Chain*** - A covalently linked biopolymer composed of specific building blocks - such as a polypeptide or protein (made of amino acids) and polynucleotide or DNA and RNA (made of nucleotides). Chains in the PDB are assigned identifiers or chain IDs, such as A, B, C so that they can be easily identified.

***Dihedral*** - This is an angle defined between 2 planes and helps position the spatial relationship between four atoms (A, B, C, D) - i.e., the angle between the plane containing atoms A, B, and C and the plane containing atoms B, C, and D.

***Entity*** - A unique molecule under investigation, and can be one of three types: polymer, non-polymer and water. Entities are described only once, even in those structures that contain multiple observations of an entity. For example, in a DNA molecule containing two identical chains the entity would be one of the individual chains rather than the double helix.

***Fo-Fc map*** - (also called a difference or omit map) Electron density map showing the difference between the observed and calculated models. I shows regions that were present in the observed map but not accounted for by the model

***2Fo-Fc map*** - Electron density map that combines the observed and omit maps (i.e., Fo + Fo-Fc). This map is commonly used for model building.

***Hydrophilic*** - Functional groups, amino acids, molecules, or domains that prefer to be in an aqueous environment are termed hydrophilic. Hydrophilic groups tend to make favorable interactions with water, usually through hydrogen bonds.

***Hydrophobic*** - Functional groups, amino acids, molecules, or domains that prefer to be in a non-aqueous (lipid) environment because they cannot make favorable interactions with water. They are usually composed of carbon and hydrogen atoms.

***Inhibitor*** - A molecule that blocks or suppresses the biological activity of another molecule.

***Ligand*** - A molecule that binds specifically to another molecule (usually a protein or nucleic acid) to form a complex. A ligand is often a small molecule, but can also be another protein.

***Molecule*** - The smallest unit of a substance that has the characteristics of that substance. A molecule consists of one or more atoms bound together.

**Non-covalent interactions**

- ***Hydrogen bonds*** - A weak bond between a hydrogen atom (that is covalently attached to one electronegative atom, such as e.g., nitrogen or oxygen) and positioned close to another electronegative atom. Hydrogen bonds are important in stabilizing many biological structures (alpha helices and beta sheets) and interactions (base pairing between complementary strands of nucleotides).
- ***Ionic bonds*** - A bond wherein two oppositely charged atoms (or ions) - one positive and one negative interact based on their charges.
- ***Van der Waals interactions*** - Weak interactions occurring between uncharged atoms due to attraction and repulsion between permanent or transient electric dipole moments. It was named after the Dutch physicist Johannes van der Waals. 
- ***Hydrophobic bonds*** - See *Hydrophobic*

***Polymer*** - A large molecule formed by covalently joining together small building blocks (monomers) e.g., amino acids and nucleotides.

***Polypeptide*** - A long string of amino acids linked by peptide bonds.

***Primary structure*** - The linear sequence of amino acids in a protein linked by peptide bonds.

***Protein symmetry*** - Point group or helical symmetry of identical subunits

- ***Protein pseudosymmetry*** - symmetry of homologous, or comparable, protein subunits
- ***Global symmetry*** - symmetry of the entire protein complex

***Quaternary structure*** - The arrangement of multiple tertiary structure domains or subunits composed of different polymer chains interacting to form higher order spatial arrangements.

***Residue*** - A single unit composing a polymer, such as an amino acid in a protein. 

***Representations*** - Molecular representation of model coordinates

- ***Ball & stick*** - A representation in which each atom is represented as a ball and all covalent interactions (bonds) are shown as sticks.
- ***Cartoon*** - The representation of backbone atoms (e.g., C-alpha atoms in proteins) shown connected by ribbons, arrows, and coils to represent secondary structures 
- ***Gaussian surface*** - A representation of the protein’s surface showing the surface of each of the atoms in the molecule 
- ***Label*** - A way to identify the amino acid residue type and amino acid number in the model. Each amino acid is shown by a label
- ***Molecular surface*** - A representation in which solvent-excluded surfaces of the molecule is shown. It shows the overall shape of the molecule, surface features and any binding pockets. 
- ***Orientation*** - A representation in which every polymer chain in the structure is shown as an oval or oblong shape, showing its orientation and overall organization of the structure.
- ***Point*** - A representation in which every unit of that component (amino acids for protein, atoms for molecule) shown by a point
- ***Putty*** - A cartoon-like representation in which the protein backbone is shown like a clay or putty structure
- ***Validation clashes*** - A representation of steric clashes between atoms in the structure. These are regions where the van der Waals shells of atoms overlap, as listed in the validation report for the structure
- ***Spacefill*** - A representation in which each atom is represented by a sphere with a radius proportional to that of the atom type (element). This representation is useful for observing how much space a molecule fills

***Root Mean Square Deviation (RMSD) of atomic position*** - Measure of the average distance between corresponding atoms in superimposed structures. A smaller value indicates similarity, while a higher value indicates divergence or difference in the structures.

***Secondary structure*** - The local arrangement of a protein arising from regular patterns of hydrogen bonding interactions within the polypeptide backbone. The two main types of secondary structure are the alpha helix and the beta-pleated sheet.

***Side chain*** - The variable group in an amino acid that is attached to the central carbon

***Tertiary structure*** - The arrangement of the secondary structure elements of a protein arising from interactions of the side chains including the formation of disulfide bonds between cysteine residues as well as non-covalent forces.

***Unit cell*** - The minimum repeat unit within a crystal which can be used to generate the entire crystal structure using only translation. The position of atoms in the unit cell are expressed in fractional coordinates in terms of a, b, and c (the axes) or as x, y, and z of cartesian coordinates. Although several different unit cells can be used for the arrangement of atoms and molecules in a crystal, the convention is to choose the unit cell with the highest symmetry.

Based on the [RCSB PDB mol* documentation](https://www.rcsb.org/3d-view/molstar/help/getting-started) thanks to the generosity of RCSB PDB and Dr. Shuchismita Dutta.
